app/experimenter/nimbus-ui/src/components/ChangeApprovalOperations/FormRemoteSettingsPending.tsx

```diff
@@ -26,19 +26,18 @@ const FormApproveConfirm = ({
 
         <div className="d-flex bd-highlight">
           <div>
-            <Button
+            <LinkExternal
               data-testid="open-remote-settings"
-              className="mr-2 btn btn-primary"
-              disabled={isLoading}
-              onClick={onConfirm}
+              className={`mr-2 btn btn-primary${isLoading ? " disabled" : ""}`}
```

Comment:
This is clever in that the `disabled` class applies `pointer-events: none` to the button so it is not clickable. What it doesn't do is prevent focusing and triggering via the keyboard. Disabling links is [tricky](https://css-tricks.com/how-to-disable-links/). We _could_ remove the href as well, but even that doesn't do the whole trick. This is probably fine as-is.

Reply:
yeah, personally I don't think maintaining custom code for that is worth it for our use case

Reply:
We could use `classNames` here, something like `className={classNames("mr-2 btn btn-primary", isLoading && "disabled"))`